samples/unit-tests/maps/drilldown/demo.js

```diff
@@ -359,3 +359,91 @@ QUnit.test('Map drilldown animation for GeoJSON maps', async assert => {
         TestUtilities.lolexUninstall(clock);
     }
 });
+
+QUnit.test(
+    '#20886, Map drilldown datalabel mouse tracking',
+    async assert => {
+        const world = await fetch(
+                'https://code.highcharts.com/mapdata/custom/world-continents.geo.json'
```

Comment:
Issues with testing .topo.json maps on drill down due to animation should be a separate issue that could be fixed if reported. I am adding this comment to find this in the future if needed.